services/provider/server/consumer.go

```diff
@@ -142,6 +144,11 @@ func (c *ocsConsumerManager) UpdateCapacity(ctx context.Context, id string, capa
 		return err
 	}
 
+	if consumerObj.Status.State == ocsv1alpha1.StorageConsumerStateDisabled || !consumerObj.Spec.Enable {
```

Comment:
@iamniting Not going to block this PR because of this, but I think we should not care about the spec, in this case, the only thing we should care about is the consumer state.